### ISSUE TITLE
Forest Stories grid rather than list

### ISSUE THREAD

**Issue by mfoucht** [ianjwhitmore/ecotrust-northwest-community-forests#10]
I know we were looking at this during our meeting, but I would still love to see these in the grid format. There is just so much white space in that list format.

**Comment by ianjwhitmore** [comment]
Still looking into this. Will keep you post today / tomorrow.

**Comment by sarahmagrish** [comment]
Site is looking soooo good! wondering about this outstanding issue—any news on what is possile?

**Comment by ianjwhitmore** [comment]
Ah not yet. Still having trouble with the theme on this one.

**Comment by ianjwhitmore** [comment]
Hey there! So... after a lot of digging, I figured out how that blog grid was being displayed.

Long story short, that version of the blog grid looks great but it didn't have the ability to paginate posts, meaning it would only show ALL or a limited number of posts without the ability to "page" through them. 

I have tried to hack a version together:
http://ianwhitmore.com/clients/ecotrust/nwcf/category/forest-stories/

It appears to be working and will work and with pagination after 6 posts display it will start to break the posts into separate pages.
